lib_eio_linux/tests/dune

```diff
@@ -1,35 +1,37 @@
+(* -*- tuareg -*- *)
+
+let linux = List.mem ("system", "linux") Jbuild_plugin.V1.ocamlc_config
```

Comment:
Note dune issue: [The documentation/value of %{system} is not consistent](https://github.com/ocaml/dune/issues/4895)

I assume this test is not more broken than `(enabled_if (= %{system} "linux"))` which, unfortunately, doesn't seem to prevent dune from failing `dune build`.

Reply:
Related issue in dune: [enabled_if doesn't work in (test)](https://github.com/ocaml/dune/issues/6938)